### PR TITLE
Update assetlist.json

### PR DESCRIPTION
**Pull request by WXRPS**
request to add WXRPS to frontier assist list so we can start our own liquidity pool
Thank you

### CODE REVIEW

File: juno/assetlist.json [hunk 1]
```diff
@@ -1218,7 +1218,28 @@
       "symbol": "CATMOS",
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/catmos.png"
-      }      
+      },
```
Comment (JeremyParish69) [inline juno/assetlist.json:1221]: missing another closing bracket

File: juno/assetlist.json [hunk 2]
```diff
@@ -1218,7 +1218,28 @@
...
+      {
+      "description": "Inspired by XRP fast network  , token Utilit) donation, memes, and gambling.",
```
Comment (JeremyParish69) [inline juno/assetlist.json:1223]: this description reads strangely...

File: juno/assetlist.json [hunk 3]
```diff
@@ -1218,7 +1218,28 @@
...
+          "denom": "WXRPS",
+          "exponent": 6
+        }
+      ],
+      "base": "cw20:juno1vwamah7e67v0rdrznxtdjrqw085v09aa7erhdagdavxeh79q60sqt4gkqc",
+      "name": "WXRPS Lighting",
+      "display": "WXRPS",
+      "symbol": "WXRPS",
+      "logo_URIs": {
+        "png": "https://images.pexels.com/photos/16466659/pexels-photo-16466659.png"
```
Comment (JeremyParish69) [inline juno/assetlist.json:1241]: png url must point to the Cosmos Chain Registry

File: juno/assetlist.json (hunk 2 above)
Comment (WXRPS) [reply juno/assetlist.json:1223]: sorry, I am way out of my head trying to get that sorted ,

"description": "Inspired by XRP Utility for replacing swift , WXRPS can be used to donation - NFT Memes - Lottery tickets "

File: juno/assetlist.json (hunk 3 above)
Comment (WXRPS) [reply juno/assetlist.json:1241]: is there a specific folder i need to upload it to ?>

Comment (JeremyParish69) [reply juno/assetlist.json:1241]: yes, juno/images/

**Comment by JeremyParish69** [comment]
gone stale

**Pull request closed without merging**